api/types/installers/agentless-installer.sh.tmpl

```diff
@@ -109,6 +109,10 @@ install_teleport() {
     sudo yum-config-manager --add-repo \
       "$(rpm --eval "https://yum.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/{{ .RepoChannel }}/teleport.repo")"
     sudo yum install -y ${PACKAGE_LIST}
+  elif echo "$ID_LIKE" | grep -q "suse"; then # ID could be -tumbleweed or -leap etc, so check ID_LIKE
+	sudo zypper addrepo "$(rpm --eval "https://yum.releases.teleport.dev/rhel/8/Teleport/%{_arch}/{{ .RepoChannel }}/teleport.repo")"
```

Comment:
This is where the `rhel/8` string is being hardcoded, I'm not sure if we'd prefer to host a full suse rpm repo that would have binaries for the proper versions of suse tumbleweed/sles/LEAP etc

Reply:
Hmm hardcoding is definitely not ideal but we don't want to have to host SUSE-specific repositories for now and use existing yum repos so this might have to do.

@fheinecke Do you have any ideas on how to handle this better or would this work for now? We could update publishing so the same repo is available via e.g. `https://yum.releases.teleport.dev/suse/15/Teleport/<arch>/<channel>/teleport.repo` (basically an alias) but not sure if it's worth it.

Reply:
@r0mant adding a separate `suse/<version>` to our publishing process as you mentioned should be trivial and will solve the issue here. How far back (how many SUSE versions) do you want us to support?

Reply:
@fheinecke Let's do 12 and 15?

Reply:
@fheinecke @r0mant  I'm not sure if we want to support it or not, but it might be also nice to have a separate `suse/tumbleweed` version in additon to 12 and 15